internal/kubernetes/operator/config_exporter.go

```diff
@@ -395,3 +402,33 @@ func (e *ConfigExporter) exportAtlasStreamProcessing(projectName string) ([]runt
 
 	return result, nil
 }
+
+func (e *ConfigExporter) exportAtlasFederatedAuth(projectName string) ([]runtime.Object, error) {
+	result := make([]runtime.Object, 0)
+	// Gets the FederationAuthSetting
+	federatedAuthentificationSetting, err := e.dataProvider.FederationSetting(&admin.GetFederationSettingsApiParams{OrgId: e.orgID})
+	if err != nil {
+		return nil, err
+	}
+	// Does not have an IdenityProvider set then no need to generate
+	if !federatedAuthentificationSetting.HasIdentityProviderStatus() || federatedAuthentificationSetting.GetIdentityProviderStatus() == "INACTIVE" {
+		return nil, err
```

Comment:
who is setting that error?

I think you might need to create the error value here instead.

Reply:
The error should bee empty. If no SAML IdP is set there is no need to export that resource.

Reply:
If the error is empty I would be explicit about it here:
```suggestion
		return nil, nil
```